verify/src/error.rs

```diff
@@ -35,8 +33,6 @@ pub enum QueryError {
 /// Contains information needed to report a good error message.
 #[derive(Debug, Clone, Serialize, PartialEq, Eq)]
 pub struct AssertionFailure {
-    /// The location of the error
-    pub loc: Span,
```

Comment:
Why did we lose the span? I'd really rather not lose this feature, unless there's a plan for bringing it back quickly in a separate PR.

Probably a destructured module needs to track spans somewhere (even if a lot of code will have to ignore them).

Reply:
Fixed.